### PR TITLE
MINOR: update log and method name

### PR DESCRIPTION
**Pull request by wcarlson5**
update an unclear log message and method name

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by wcarlson5** [comment]
@guozhangwang Some changes you requested in a follow up PR

**Review by guozhangwang — APPROVED** [review]
Thanks @wcarlson5 !

**Comment by guozhangwang** [comment]
Re-trigger the jenkins jobs.

**Comment by ableegoldman** [comment]
Tests were still failing despite Guozhang having retriggered them after the fix was merged, I don't see how this patch could be causing those tests to fail again but I retriggered the build one more time just to be sure

@wcarlson5 please ping me or Guozhang again when it passes and we can merge

**Review by ableegoldman — APPROVED** [review]
LGTM

File: streams/src/main/java/org/apache/kafka/streams/processor/internals/TopologyMetadata.java [hunk 1]
```diff
@@ -153,7 +153,7 @@ public void maybeNotifyTopologyVersionWaiters(final String threadName) {
                     if (threadVersions.values().stream().allMatch(t -> t >= topologyVersionWaitersVersion)) {
                         topologyVersionWaiters.future.complete(null);
                         iterator.remove();
-                        log.info("Thread {} is now on topology version {}", threadName, topologyVersionWaiters.topologyVersion);
+                        log.info("All threads are now on topology version {}", topologyVersionWaiters.topologyVersion);
```
Comment (showuon) [inline streams/src/main/java/org/apache/kafka/streams/processor/internals/TopologyMetadata.java:156]: Sorry, some comments to this change:
I can imagine after this PR, there will be some log output in `maybeNotifyTopologyVersionWaitersAndUpdateThreadsTopologyVersion` like this (if the `version.activeTopologyWaiters` has 3 waiters):
```
[2021-12-15 08:52:59,397] INFO All threads are now on topology version 1
[2021-12-15 08:52:59,397] INFO All threads are now on topology version 1
[2021-12-15 08:52:59,397] INFO All threads are now on topology version 1
...
```
I don't think the log is helpful. Could we add some other info here? Like: `All threads are now on topology version {}, removing the waiter {}, topologyVersionWaiters.topologyVersion, topologyVersionWaiters.future)`. WDYT?

**Review by showuon — COMMENTED** [review]
@wcarlson5 , thanks for the PR. Left a comment. Thanks.

**Review by showuon — COMMENTED** [review]
Also, please rebase to the latest trunk, all the failed tests will be passed. Thanks.

File: streams/src/main/java/org/apache/kafka/streams/processor/internals/TopologyMetadata.java (hunk 1 above)
Comment (wcarlson5) [reply streams/src/main/java/org/apache/kafka/streams/processor/internals/TopologyMetadata.java:156]: this log line only prints once not once per thread. each waiter also has a different version

Comment (showuon) [reply streams/src/main/java/org/apache/kafka/streams/processor/internals/TopologyMetadata.java:156]: I see. Thank you.

**Review by showuon — APPROVED** [review]
LGTM

**Comment by ableegoldman** [comment]
Test failures are unrelated:
kafka.server.ReplicaManagerTest.[1] usesTopicIds=true 
kafka.api.PlaintextConsumerTest.testPerPartitionLagMetricsCleanUpWithSubscribe()

**Comment by ableegoldman** [comment]
Merged to trunk

**Pull request merged**